modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -636,6 +636,85 @@ definition {
 		}
 	}
 
+	@description = "LPS-178410 - Verify if actions works with oldValue function when trigger is On After Update (Notification) in First Name field"
```

Comment:
Ticket number should be 178415

Reply:
Instead of "First Name field" replace with User System Object

Reply:
Oh yeah! I will be more careful in checking this informations in the ticket.